### PR TITLE
Prql-js

### PR DESCRIPTION
**Pull request by aljazerzen**
@clifton the package you've publish on npm does not seem to work for me, So I've packaged prql-js myself as you described on discord.

Can you take a look if it looks ok and do a republish?

### CODE REVIEW

**Comment by max-sixty** [comment]
(FYI I think @clifton is away this week but I'm sure will respond when he's back)

File: prql-js/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+language: rust
```
Comment (max-sixty) [inline prql-js/.travis.yml]: We can remove this (& the appveyor file too) I'm guessing

**Comment by clifton** [comment]
yes sure, just got home! one thing is that npm forces a version bump for each publish (and `wasm-pack` uses the version specified in the `Cargo.toml` file)

**Comment by clifton** [comment]
@aljazerzen i added you as a maintainer to [prql-js on npm](https://www.npmjs.com/package/prql-js), so you should be able to publish if you bump the version

**Comment by aljazerzen** [comment]
Published:
https://www.npmjs.com/package/prql-js

**Pull request merged**